app/views/staff/organisations/_grouped_programmes.html.haml

```diff
@@ -0,0 +1,28 @@
+- if grouped_programmes.any?
+  .govuk-grid-row
+    .govuk-grid-column-full
+      %h2.govuk-heading-m
+        = t("page_content.organisation.programmes")
+
+      - grouped_programmes.each do |fund, activities|
+        - if activities.any?
+          %table{id: fund.id, class: "govuk-table"}
+            %caption{class: "govuk-table__caption govuk-table__caption--s"}
```

Comment:
How does `govuk-table__caption--m` look with this? First instinct when looking at the screenshot was the text was a smidge too small

Reply:
I was going by the design and it's one step down from the medium-sized `<h2>`.

Reply:
And it turns out that `govuk-table__caption--m` doesn't exist in the outdated version of `govuk-frontend` we've been using for a while... 🐰 🕳️

Reply:
Have amended the caption, adding some spacing and fixed the width of the "title" column so the fund tables both line up:

![Screenshot 2021-02-17 at 12 33 39](https://user-images.githubusercontent.com/3166/108204984-61470780-711c-11eb-8b47-4e1b899a3efb.png)

I've also changed "Identifier" to "RODA identifier" to be consistent with the rest of the site.